### ISSUE TITLE
PrimaryColumns other than 'id' are not supported in codegen

### ISSUE THREAD

**Issue by jeffrey-computers** [subsquid/squid-sdk#259]
**Describe the bug**
TypeORM `@PrimaryColumn` supports fields with an arbitrary name, but Squid's codegen only supports a PrimaryColumn named 'id'. If an 'id' value is not included in the schema, then it is added automatically.

**To Reproduce**
Create a `schema.graphql` file like:
```
type TestEntity @entity {
  someUniqueIdentifier: ID!
  foo: boolean;
}
```

Codegen automatically creates a `@PrimaryColumn_()` of `id`. The intended primary column is `someUniqueIdentifier`

**Expected behavior**
- Schema entity fields typed as `ID!` should be used for TypeORM's `PrimaryColumn` instead of id.
- No `id` field should be created unless an `ID!` is missing.

i.e. the above example
Should yield:

```js
@Entity_()
export class TestEntity {
    constructor(props?: Partial<TestEntity>) {
        Object.assign(this, props)
    }

    @PrimaryColumn_()
    someUniqueIdentifier!: string

    @Column_("bool", {nullable: true})
    foo!: boolean | undefined | null
}
```

**Comment by rmcmk** [comment]
I agree. Primary keys should not be restricted in this manner. Additionally, supporting composite primary keys would allow for more flexible indexing paradigms.

However, this may not be a trivial change. The code generator adheres to the OpenCRUD spec, which currently only supports primary keys of type `ID` (essentially a string) and does not support composite primary keys (maybe it does, am not completely familiar with the spec)

**Comment by sandrocsimas** [comment]
Does it work if I change the model manually? Or is there anything else inside the crud operations that does not allow me to modify the ID?

**Comment by sandrocsimas** [comment]
Never mind, hehe. I see a lot of places using `e.id`. Here, for example:
https://github.com/subsquid/squid-sdk/blob/master/typeorm/typeorm-store/src/hot.ts
